examples/bri-3/components/storage/sql-connection.js

```diff
@@ -0,0 +1,79 @@
+const { Client } = require('pg');
+
+//Terminal cmd to spin up pg docker image
+//docker run --name postgres -e POSTGRES_PASSWORD=example -p 5433:5432 -d postgres 
+
+//Local DB
+// const client = new Client({
+//     user: 'postgres',
+//     host: 'localhost',
+//     database: 'postgres',
+//     password: '',
+//     port: 5432,
+// });
+
+//Docker DB
+const client = new Client({
+    user: 'postgres',
+    host: 'localhost',
+    database: 'postgres',
+    password: 'example',
+    port: 5433,
+});
+
+client.connect();
+
+const ordersTable = `CREATE TABLE orders (
+    id int,
+    price decimal,
+    created_at timestamp
+);`;
+
+const bpiTable = `CREATE TABLE bpi (
+    id int,
+    name varchar,
+    product_id int,
+    created_at timestamp,
+    last_updated timestamp
+);`;
+
+const workstepsTable = `CREATE TABLE worksteps (
+    id int,
+    name varchar,
+    version varchar
+);`;
+
+const bpiTimestampRecordsTable = `CREATE TABLE bpiTimestampRecords (
+    id int,
+    bpi_id int,
+    updated timestamp 
+);`;
+
+const stateTable = `CREATE TABLE state (
+    id int, --[pk, increment]
+    agreementState varchar,
+    acceptanceStatus varchar
+);`;
+
+const tables = [ordersTable, bpiTable, workstepsTable, bpiTimestampRecordsTable, stateTable]
+
+//run a query to run the create tables listed above, included in the tables array
+tables.forEach(table => {
```

Comment:
when init migration is there this is redundant

Reply:
Yes, all the entities will be added through  prisma.schema and then migrations (or a init migration) generated though prisma commands. This migration\s will then be executed by the app on startup. This file is not needed.

Reply:
Agreed. @ognjenkurtic could you please validate if the default ORM setting to "always" spin-up tables is disabled?

Reply:
It is not setup atm to do anything when the app starts. We manually trigger db upgrades with prisma cli.